tekton-pipeline/pipeline-jenkins-plugins.yaml

```diff
@@ -0,0 +1,64 @@
+﻿apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  creationTimestamp: '2020-10-21T07:43:27Z'
+  generation: 5
+  managedFields:
+    - apiVersion: tekton.dev/v1beta1
+      fieldsType: FieldsV1
+      fieldsV1:
+        'f:spec':
+          .: {}
+          'f:params': {}
+          'f:resources': {}
+          'f:tasks': {}
+      manager: Mozilla
+      operation: Update
+      time: '2020-10-21T13:48:54Z'
+  name: jenkins-plugins
+  namespace: art-jenkins
+  resourceVersion: '69266116'
+  selfLink: /apis/tekton.dev/v1beta1/namespaces/art-jenkins/pipelines/jenkins-plugins
+  uid: 4242aba0-7556-4231-9610-7ea855e899ee
+spec:
+  params:
+    - default: >-
+        https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_jenkins/1153/pull-ci-openshift-jenkins-release-4.4-images/1306572105390755840/artifacts/build-logs/jenkins.log
+      description: upstream url of jenkins log
+      name: build-url
+      type: string
```

Comment:
wait... this is really where we get the plugin list from??

Reply:
en..I found it's really hard to handle a long list of plugins,they are out of range as a string, seems they don't have much kinds of type to use for parameter. So I use a trick as I saw in the requested JIRA, the jenkins plugins list extract from jenkins update log. [ART-2254](https://issues.redhat.com/browse/ART-2254)